### ISSUE TITLE
Cannot switch to Windows Containers with Docker for Windows

### ISSUE THREAD

**Issue by xalaliah** [docker/for-win#4028]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: B209ECEA-4431-4902-B885-DE4014C7B750/20190603161609

### Expected behavior
Docker switches from using Linux Containers to using Windows Containers

### Actual behavior
Docker is stuck "switching..." for several minutes before failing with following output: 
```
[16:09:49.048][DockerDaemonChecker][Info   ] Docker daemon is running
[16:09:49.050][SegmentApi     ][Info   ] Usage statistic: eventBackendStarted
[16:09:49.048][NamedPipeServer][Info   ] Restart done in 00:01:17.5899613.
[16:09:49.053][Notifications  ][Info   ] Docker Desktop is running
[16:09:54.558][Moby           ][Info   ] [   46.953161] nf_conntrack: default automatic helper assignment has been turned off for security reasons and CT-based  firewall rule not found. Use the iptables CT target to attach helpers instead.
[16:10:00.809][Moby           ][Info   ] [   53.241097] hv_balloon: Received INFO_TYPE_MAX_PAGE_CNT
[16:10:00.821][Moby           ][Info   ] [   53.251092] hv_balloon: Data Size is 8
[16:10:04.906][SegmentApi     ][Info   ] Usage statistic: actionMenuSwitchWindowsDaemon
[16:10:04.908][Notifications  ][Info   ] Docker Desktop is switching...
[16:10:04.923][NamedPipeClient][Info   ] Sending Version()...
[16:10:04.925][NamedPipeClient][Info   ] Received response for Version
[16:10:04.925][NamedPipeClient][Info   ] Sending Start(Windows, Docker.Core.Settings, Docker.Core.DaemonOptions)...
[16:10:04.924][NamedPipeServer][Info   ] Version()
[16:10:04.924][NamedPipeServer][Info   ] Version done in 00:00:00.
[16:10:04.926][NamedPipeServer][Info   ] Start(Windows, Docker.Core.Settings, Docker.Core.DaemonOptions)
[16:10:05.221][WindowsDaemon  ][Warning] Running experimental build
[16:10:05.226][WindowsDaemon  ][Info   ] debug: Listener created for HTTP on npipe (//./pipe/docker_engine_windows)
[16:10:05.227][WindowsDaemon  ][Info   ] debug: Service running
[16:10:05.234][WindowsDaemon  ][Info   ] debug: Stackdump - waiting signal at Global\docker-daemon-14204
[16:10:05.234][WindowsDaemon  ][Info   ] Windows default isolation mode: hyperv
[16:10:05.237][WindowsDaemon  ][Info   ] debug: Using default logging driver json-file
[16:10:05.237][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: windowsfilter
[16:10:05.237][WindowsDaemon  ][Info   ] debug: WindowsGraphDriver InitFilter at C:\ProgramData\Docker\windowsfilter
[16:10:05.238][WindowsDaemon  ][Info   ] debug: Initialized graph driver windowsfilter
[16:10:05.239][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: lcow
[16:10:05.242][WindowsDaemon  ][Info   ] lcowdriver: init: dataRoot: C:\ProgramData\Docker\lcow globalMode: false
[16:10:05.243][WindowsDaemon  ][Info   ] debug: Initialized graph driver lcow
[16:10:05.346][WindowsDaemon  ][Info   ] debug: Max Concurrent Downloads: 3
[16:10:05.350][WindowsDaemon  ][Info   ] debug: Max Concurrent Uploads: 5
[16:10:05.351][WindowsDaemon  ][Info   ] Loading containers: start.
[16:10:05.351][WindowsDaemon  ][Info   ] debug: Option Experimental: true
[16:10:05.352][WindowsDaemon  ][Info   ] debug: Option DefaultDriver: nat
[16:10:05.352][WindowsDaemon  ][Info   ] debug: Option DefaultNetwork: nat
[16:10:05.352][WindowsDaemon  ][Info   ] debug: Network Control Plane MTU: 1500
[16:10:05.352][WindowsDaemon  ][Info   ] Restoring existing overlay networks from HNS into docker
[16:10:05.352][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:10:06.171][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:10:08.770][WindowsDaemon  ][Info   ] debug: daemon configured with a 15 seconds minimum shutdown timeout
[16:10:08.770][WindowsDaemon  ][Info   ] debug: start clean shutdown of all containers with a 15 seconds timeout...
[16:10:08.775][WindowsDaemon  ][Error  ] fatal: Error starting daemon: Error initializing network controller: hnsCall failed in Win32: No such interface supported (0x80004002)
[16:11:08.865][WindowsDaemon  ][Warning] Running experimental build
[16:11:08.870][WindowsDaemon  ][Info   ] debug: Listener created for HTTP on npipe (//./pipe/docker_engine_windows)
[16:11:08.871][WindowsDaemon  ][Info   ] debug: Service running
[16:11:08.877][WindowsDaemon  ][Info   ] Windows default isolation mode: hyperv
[16:11:08.877][WindowsDaemon  ][Info   ] debug: Stackdump - waiting signal at Global\docker-daemon-6440
[16:11:08.880][WindowsDaemon  ][Info   ] debug: Using default logging driver json-file
[16:11:08.880][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: windowsfilter
[16:11:08.880][WindowsDaemon  ][Info   ] debug: WindowsGraphDriver InitFilter at C:\ProgramData\Docker\windowsfilter
[16:11:08.881][WindowsDaemon  ][Info   ] debug: Initialized graph driver windowsfilter
[16:11:08.882][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: lcow
[16:11:08.884][WindowsDaemon  ][Info   ] lcowdriver: init: dataRoot: C:\ProgramData\Docker\lcow globalMode: false
[16:11:08.884][WindowsDaemon  ][Info   ] debug: Initialized graph driver lcow
[16:11:09.009][WindowsDaemon  ][Info   ] debug: Max Concurrent Downloads: 3
[16:11:09.010][WindowsDaemon  ][Info   ] debug: Max Concurrent Uploads: 5
[16:11:09.010][WindowsDaemon  ][Info   ] Loading containers: start.
[16:11:09.011][WindowsDaemon  ][Info   ] debug: Option Experimental: true
[16:11:09.011][WindowsDaemon  ][Info   ] debug: Option DefaultDriver: nat
[16:11:09.011][WindowsDaemon  ][Info   ] debug: Option DefaultNetwork: nat
[16:11:09.012][WindowsDaemon  ][Info   ] debug: Network Control Plane MTU: 1500
[16:11:09.013][WindowsDaemon  ][Info   ] Restoring existing overlay networks from HNS into docker
[16:11:09.013][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:11:09.819][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:11:10.440][WindowsDaemon  ][Info   ] debug: daemon configured with a 15 seconds minimum shutdown timeout
[16:11:10.440][WindowsDaemon  ][Info   ] debug: start clean shutdown of all containers with a 15 seconds timeout...
[16:11:10.444][WindowsDaemon  ][Error  ] fatal: Error starting daemon: Error initializing network controller: hnsCall failed in Win32: The object invoked has disconnected from its clients. (0x80010108)
[16:12:10.502][WindowsDaemon  ][Warning] Running experimental build
[16:12:10.505][WindowsDaemon  ][Info   ] debug: Listener created for HTTP on npipe (//./pipe/docker_engine_windows)
[16:12:10.514][WindowsDaemon  ][Info   ] debug: Service running
[16:12:10.514][WindowsDaemon  ][Info   ] Windows default isolation mode: hyperv
[16:12:10.514][WindowsDaemon  ][Info   ] debug: Stackdump - waiting signal at Global\docker-daemon-9412
[16:12:10.515][WindowsDaemon  ][Info   ] debug: Using default logging driver json-file
[16:12:10.515][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: windowsfilter
[16:12:10.516][WindowsDaemon  ][Info   ] debug: WindowsGraphDriver InitFilter at C:\ProgramData\Docker\windowsfilter
[16:12:10.517][WindowsDaemon  ][Info   ] debug: Initialized graph driver windowsfilter
[16:12:10.519][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: lcow
[16:12:10.521][WindowsDaemon  ][Info   ] lcowdriver: init: dataRoot: C:\ProgramData\Docker\lcow globalMode: false
[16:12:10.521][WindowsDaemon  ][Info   ] debug: Initialized graph driver lcow
[16:12:10.614][WindowsDaemon  ][Info   ] debug: Max Concurrent Downloads: 3
[16:12:10.614][WindowsDaemon  ][Info   ] debug: Max Concurrent Uploads: 5
[16:12:10.614][WindowsDaemon  ][Info   ] Loading containers: start.
[16:12:10.615][WindowsDaemon  ][Info   ] debug: Option Experimental: true
[16:12:10.615][WindowsDaemon  ][Info   ] debug: Option DefaultDriver: nat
[16:12:10.615][WindowsDaemon  ][Info   ] debug: Option DefaultNetwork: nat
[16:12:10.616][WindowsDaemon  ][Info   ] debug: Network Control Plane MTU: 1500
[16:12:10.616][WindowsDaemon  ][Info   ] Restoring existing overlay networks from HNS into docker
[16:12:10.616][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:12:11.578][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:12:14.181][WindowsDaemon  ][Info   ] debug: daemon configured with a 15 seconds minimum shutdown timeout
[16:12:14.181][WindowsDaemon  ][Info   ] debug: start clean shutdown of all containers with a 15 seconds timeout...
[16:12:14.186][WindowsDaemon  ][Error  ] fatal: Error starting daemon: Error initializing network controller: hnsCall failed in Win32: No such interface supported (0x80004002)
[16:13:06.890][Cmd            ][Info   ] time="2019-06-03T16:13:06-04:00" level=fatal msg="The specified service already exists."
[16:13:06.957][WindowsDaemon  ][Warning] Running experimental build
[16:13:06.960][WindowsDaemon  ][Info   ] debug: Listener created for HTTP on npipe (//./pipe/docker_engine_windows)
[16:13:06.961][WindowsDaemon  ][Info   ] debug: Service running
[16:13:06.970][WindowsDaemon  ][Info   ] Windows default isolation mode: hyperv
[16:13:06.970][WindowsDaemon  ][Info   ] debug: Stackdump - waiting signal at Global\docker-daemon-13712
[16:13:06.971][WindowsDaemon  ][Info   ] debug: Using default logging driver json-file
[16:13:06.971][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: windowsfilter
[16:13:06.972][WindowsDaemon  ][Info   ] debug: WindowsGraphDriver InitFilter at C:\ProgramData\Docker\windowsfilter
[16:13:06.972][WindowsDaemon  ][Info   ] debug: Initialized graph driver windowsfilter
[16:13:06.974][WindowsDaemon  ][Info   ] debug: [graphdriver] trying provided driver: lcow
[16:13:06.975][WindowsDaemon  ][Info   ] lcowdriver: init: dataRoot: C:\ProgramData\Docker\lcow globalMode: false
[16:13:06.975][WindowsDaemon  ][Info   ] debug: Initialized graph driver lcow
[16:13:07.109][WindowsDaemon  ][Info   ] debug: Max Concurrent Downloads: 3
[16:13:07.109][WindowsDaemon  ][Info   ] debug: Max Concurrent Uploads: 5
[16:13:07.109][WindowsDaemon  ][Info   ] Loading containers: start.
[16:13:07.110][WindowsDaemon  ][Info   ] debug: Option Experimental: true
[16:13:07.110][WindowsDaemon  ][Info   ] debug: Option DefaultDriver: nat
[16:13:07.110][WindowsDaemon  ][Info   ] debug: Option DefaultNetwork: nat
[16:13:07.110][WindowsDaemon  ][Info   ] debug: Network Control Plane MTU: 1500
[16:13:07.111][WindowsDaemon  ][Info   ] Restoring existing overlay networks from HNS into docker
[16:13:07.111][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:13:07.866][WindowsDaemon  ][Info   ] debug: [GET]=>[/networks/] Request : 
[16:13:10.455][WindowsDaemon  ][Info   ] debug: daemon configured with a 15 seconds minimum shutdown timeout
[16:13:10.455][WindowsDaemon  ][Info   ] debug: start clean shutdown of all containers with a 15 seconds timeout...
[16:13:10.459][WindowsDaemon  ][Error  ] fatal: Error starting daemon: Error initializing network controller: hnsCall failed in Win32: No such interface supported (0x80004002)
[16:13:43.717][SegmentApi     ][Info   ] Usage statistic: heartbeat
[16:16:07.400][ApiProxy       ][Info   ] Killing existing com.docker.proxy with PID 14164
[16:16:07.589][NamedPipeClient][Error  ] Unable to send Start: fatal: Error starting daemon: Error initializing network controller: hnsCall failed in Win32: No such interface supported (0x80004002)
[16:16:07.589][NamedPipeServer][Error  ] Unable to execute Start: fatal: Error starting daemon: Error initializing network controller: hnsCall failed in Win32: No such interface supported (0x80004002)    at Docker.Backend.Processes.WindowsDockerDaemon.WaitForDaemonToBeReady() in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Backend\Processes\WindowsDockerDaemon.cs:line 193
   at Docker.Backend.ContainerEngine.Windows.DoStart(Settings settings, String daemonOptions) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Backend\ContainerEngine\Windows.cs:line 225
   at Docker.Backend.ContainerEngine.Windows.Start(Settings settings, String daemonOptions) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Backend\ContainerEngine\Windows.cs:line 98
   at Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass9_0.<Register>b__0(Object[] parameters) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Core\pipe\NamedPipeServer.cs:line 48
   at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Core\pipe\NamedPipeServer.cs:line 180
[16:16:07.589][NamedPipeClient][Info   ] Sending Version()...
[16:16:07.589][NamedPipeClient][Info   ] Received response for Version
[16:16:07.589][NamedPipeClient][Info   ] Sending Start(Linux, Docker.Core.Settings, Docker.Core.DaemonOptions)...
[16:16:07.589][NamedPipeServer][Info   ] Version()
[16:16:07.589][NamedPipeServer][Info   ] Version done in 00:00:00.
[16:16:07.589][NamedPipeServer][Info   ] Start(Linux, Docker.Core.Settings, Docker.Core.DaemonOptions)
[16:16:07.589][PowerShell     ][Info   ] Run script '(Hyper-V\Get-VM MobyLinuxVM).Id.Guid'...
[16:16:07.872][DockerDaemonChecker][Error  ] Docker daemon is not running
[16:16:07.935][ApiProxy       ][Info   ] time="2019-06-03T16:16:07-04:00" msg="docker proxy: ready"
[16:16:07.937][ApiProxy       ][Info   ] time="2019-06-03T16:16:07-04:00" msg="🍀 socket server starting : \\\\.\\pipe\\dockerGuiToDriver"
[16:16:09.097][ApiProxy       ][Info   ] time="2019-06-03T16:16:09-04:00" msg="proxy >> GET /_ping\n"
[16:16:09.099][ApiProxy       ][Info   ] time="2019-06-03T16:16:09-04:00" msg="proxy << GET /_ping (2.0031ms)\n"
[16:16:09.101][ApiProxy       ][Info   ] time="2019-06-03T16:16:09-04:00" msg="proxy >> GET /v1.39/containers/json\n"
[16:16:09.102][ApiProxy       ][Info   ] time="2019-06-03T16:16:09-04:00" msg="proxy << GET /v1.39/containers/json (2.005ms)\n"
[16:16:09.108][NamedPipeClient][Info   ] Received response for Start
[16:16:09.110][SegmentApi     ][Info   ] Usage statistic: eventBackendStarted
[16:16:09.108][DockerDaemonChecker][Info   ] Docker daemon is running
[16:16:09.108][NamedPipeServer][Info   ] Start done in 00:00:01.5193901.
[16:16:09.113][Notifications  ][Info   ] Docker Desktop is running
[16:16:09.115][Notifications  ][Error  ] fatal: Error starting daemon: Error initializing network controller: hnsCall failed in Win32: No such interface supported (0x80004002)
[16:16:09.122][NamedPipeClient][Info   ] Sending Version()...
[16:16:09.124][NamedPipeClient][Info   ] Received response for Version
[16:16:09.124][NamedPipeClient][Info   ] Sending GatherDiagnostics(-path C:\Users\19030430\AppData\Local\Temp\ gather -id B209ECEA-4431-4902-B885-DE4014C7B750/20190603161609, C:\Users\19030430\AppData\Local\Temp\B209ECEA-4431-4902-B885-DE4014C7B750/20190603161609.zip)...
[16:16:09.123][NamedPipeServer][Info   ] Version()
[16:16:09.123][NamedPipeServer][Info   ] Version done in 00:00:00.
[16:16:09.125][NamedPipeServer][Info   ] GatherDiagnostics(-path C:\Users\19030430\AppData\Local\Temp\ gather -id B209ECEA-4431-4902-B885-DE4014C7B750/20190603161609, C:\Users\19030430\AppData\Local\Temp\B209ECEA-4431-4902-B885-DE4014C7B750/20190603161609.zip)
[16:16:17.793][ErrorReportWindow][Info   ] Open logs
```

### Information

  - Originally, I had Docker up and working fine running swarm with Hyper-V. This problem occurred after I moved from Ethernet connection to a wireless connection (for right now, I need to have this set up and working on WiFi. Later on, I'll need docker configured for Ethernet again. There's nothing I can do otherwise).
- I have fully uninstalled Docker, disabled Hyper-v, removed the DockerNAT adapter, restarted desktop, reinstalled Docker, reenabled Hyper-v, and restarted desktop. No effect. 
- I have fully uninstalled Docker,  C:\Program Files\Docker\Docker\  , deleted C:\ProgramData\Docker\ , deleted C:\ProgramData\DockerDesktop\ , and reinstalled Docker. No effect.
- I have run WindowsContainerNetworking-LoggingAndCleanupAide.ps1 with both the -Cleanup and -ForceDeleteAllSwitches. Restart. No effect.
- I have manually disabled and uninstalled all manually created virtual switches and adapters. No effect.
- Reset Network Settings. No effect.
- Reset Docker to factory settings. No effect.
- Flushed TCP/IP. No effect.
- Get-NetNat and Get-HNSNetwork show nothing, but Get-VMSwitch does show my external vswitch:
```
PS C:\Users\19030430> get-netnat
PS C:\Users\19030430> get-hnsnetwork

PS C:\Users\19030430> get-vmswitch

Name           SwitchType NetAdapterInterfaceDescription
----           ---------- ------------------------------
extswitch      External   Intel(R) Centrino(R) Advanced-N 6235
DockerNAT      Internal
Default Switch Internal
```

I ran this scriptlet from [Window's Container Troubleshooting](https://docs.microsoft.com/en-us/virtualization/windowscontainers/troubleshooting) :
`Invoke-WebRequest https://aka.ms/Debug-ContainerHost.ps1 -UseBasicParsing | Invoke-Expression`
which gave me the following output:
```
Checking for common problems
Container Host OS Product Name: Windows 10 Enterprise
Container Host OS Build Label: 17763.1.amd64fre.rs5_release.180914-1434
Describing Windows Version and Prerequisites
 [+] Is Windows 10 Anniversary Update or Windows Server 2016 630ms
 [+] Has KB3192366, KB3194496, or later installed if running Windows build 14393 132ms
 [+] Is not a build with blocking issues 30ms
 [+] Has 'Containers' feature installed 47.64s
Describing Docker is installed
 [+] A Docker service is installed - 'Docker' or 'com.Docker.Service'  73ms
 [+] Service is running 29ms
 [+] Docker.exe is in path 5.23s
 [+] Docker is registered in the EventLog service 85ms
Describing User has permissions to use Docker daemon
 [+] docker.exe should not return access denied 37ms
Describing Windows container settings are correct
 [+] Do not have DisableVSmbOplock set to 1 29ms
 [+] Do not have zz values set 30ms
 [+] Do not have FDVDenyWriteAccess set to 1 41ms
Describing The right container base images are installed
WARNING: No mcr.microsoft.com/* base images found. Checking for deprecated images.
 [-] At least one of 'mcr.microsoft.com/windows/servercore', 'mcr.microsoft.com/windows/nanoserver', 'mcr.microsoft.com/windows' or deprecated microsoft/windowsservercore, microsoft/nanoserver should be installed 580ms
   ValidationMetadataException: The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
   ParameterBindingValidationException: Cannot validate argument on parameter 'Property'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
   at <ScriptBlock>, <No file>: line 126
Describing Container network is created
 [+] At least one local container network is available 2.18s
 [-] At least one NAT, Transparent, or L2Bridge Network exists 94ms
   Expected {0} to be greater than {0}
   218:       $totalnets | Should BeGreaterThan 0
   at <ScriptBlock>, <No file>: line 218
 [-] NAT Network's vSwitch is internal 58ms
   Expected: {Internal}
   But was:  {}
   223:       $switchType | Should Be "Internal"
   at <ScriptBlock>, <No file>: line 223
 [+] A Windows NAT is configured if a Docker NAT network exists 70ms
 [-] Specified Network Gateway IP for NAT network is assigned to Host vNIC 51ms
   Expected: value to not be empty
   237:       $natGatewayIP | Should Not BeNullOrEmpty
   at <ScriptBlock>, <No file>: line 237
 [-] NAT Network's internal prefix does not overlap with external IP' 40ms
   Expected {0} to be greater than {0}
   264:         $hostips.Count | Should BeGreaterThan 0
   at <ScriptBlock>, <No file>: line 264
Showing output from: docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 0
Server Version: 18.09.2
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 9754871865f7fe2f4e74d43e2fc7ccd237edcbce
runc version: 09c8266bf2fcf9519a651b04ae54c967b9ab86ec
init version: fec3683
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.125-linuxkit
Operating System: Docker for Windows
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.934GiB
Name: linuxkit-00155d2a1101
ID: GM2E:U6NM:FA6P:JZUW:M6M3:7YLY:26ZA:PCSG:4KM4:23XK:ALED:J66H
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 23
 Goroutines: 47
 System Time: 2019-06-03T20:46:35.9197359Z
 EventsListeners: 1
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Product License: Community Engine


Showing output from: docker version
Client: Docker Engine - Community
 Version:           18.09.2
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        6247962
 Built:             Sun Feb 10 04:12:31 2019
 OS/Arch:           windows/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       6247962
  Built:            Sun Feb 10 04:13:06 2019
  OS/Arch:          linux/amd64
  Experimental:     false

Showing output from: docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
8890c7123dea        bridge              bridge              local
8fd11cbb3d35        host                host                local
2057fe057ad3        none                null                local

Getting Warnings & errors in the Windows event logs from the last 24 hours
Logs saved to C:\Users\19030430\logs_20190603-164644.csv


Getting Docker for Windows daemon logs from the last execution
    Note: More logs may be available at C:\Users\19030430\AppData\Local\Docker. Only showing the latest 100 lines.
```

  - Windows Version: Windows 10 Enterprise Version 1809
  - Docker for Windows Version: 18.09.2

### Steps to reproduce the behavior

  1. Use Docker with Windows Containers as normal while connected to Ethernet. Create some docker-machines, start up a swarm and some services, etc etc.
  2. Switch from Ethernet connection to wireless
  3. Try to use Windows Containers

**Comment by xalaliah** [comment]
I'll have to admit, I have no idea what exactly went wrong here. I tried for many more days to resolve this issue. I ended up having to re image Windows 10 on my laptop to start out completely fresh. Upon re-imaging I found that my registry had a lot of skeletons that needed to be cleaned out, but I'm pretty sure this was from me messing with all of my network and adapter settings after I started getting this error.
Seems to me that no one else has had this issue. I'll have to assume it was an anomaly within my setup.
I'll reopen if this occurs when I need to switch back from Wi-Fi to Ethernet.
